cli/src/tasks/sourcemaps.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  readdirSync,
+  existsSync,
+  readFileSync,
+  writeFileSync,
+  unlinkSync,
+} from '@ionic/utils-fs';
+import { join, extname } from 'path';
+
+import type { Config } from '../definitions';
+import { logger } from '../log';
+
+function findJSAssetsDir(buildDir: string): string {
+  const reactJSDir = '/static/js';
+  const vueJSDir = '/js';
```

Comment:
Path.join is required for folder separators which will change based on platform (eg on windows its \ rather than /)